Gobabygo/Gobabygo.ino

```diff
@@ -31,25 +31,27 @@ void setup() {
 
 void loop() {
   // put your main code here, to run repeatedly:
-  float joyXVal;
+  int joyXVal;
   float scaledJoyXVal;
   int joyYVal;
   float scaledJoyYVal;
   
   float rightMotorVal;
   float leftMotorVal;
   float scaler;
+
+  const float backwardPower = 0.4;
```

Comment:
Suggest to declare this above with other constants like dead zone

Reply:
We can think of the top of the file as being same as our Constants.java. Having them above makes it easy to tune/ tweak them easily.